test/.eslintrc

```diff
@@ -0,0 +1,3 @@
+env:
```

Comment:
One set it for all files, instead of having to add the boilerplate to each single test file. As atm we have 2 test files, this was the DRYest strategy. Thats why I went this way